services/appflowy-collaborate/src/group/broadcast.rs

```diff
@@ -105,6 +115,14 @@ impl CollabBroadcast {
             event.update.len(),
             origin
           );
+
+          let stream_update = event.update.clone();
+          let cloned_update_streaming = update_streaming.clone();
+          // FIXME: how to maintain the order of the update when using spawn here.
+          tokio::spawn(async move {
+            cloned_update_streaming.send_update(stream_update).await;
```

Comment:
Instead of using spawn, use a queue ie. `tokio::sync::mpsc::UnboundedSender` and send the updates through it. Then spawn a tokio task which will consume incoming updates from `UnboundedReceiver` on the other end.

Reply:
Agreed. I used this approach in client-api. It will merge multiple updates within a period of time.